musicbot/bot.py

```diff
@@ -1696,12 +1696,12 @@ async def _cmd_play(
         song_url,
         head,
     ):
-        if _player:
-            player = _player
-        elif permissions.summonplay:
-            vc = author.voice.channel if author.voice else None
+        player = _player if _player else None
```

Comment:
This is the actual fix. `player` either set to the input param or None. Previously it could be undefined, which lead to the exception in line 1719.